config/environments/development.rb

```diff
@@ -11,7 +11,7 @@
 
   # Show full error reports and enable caching
   config.consider_all_requests_local       = true
-  config.action_controller.perform_caching = true
+  config.action_controller.perform_caching = false
```

Comment:
I can see why you might do this temporarily but not sure if it's a good idea to commit it. Perhaps put it in your local_config.rb?